src/lib/routes/admin-api/project/features.ts

```diff
@@ -72,21 +91,61 @@ export default class ProjectFeaturesController extends Controller {
         this.featureService = featureToggleServiceV2;
         this.logger = config.getLogger('/admin-api/project/features.ts');
 
-        this.get(`${PATH_ENV}`, this.getEnvironment);
+        this.route({
+            method: 'get',
+            path: PATH_ENV,
+            acceptAnyContentType: true,
+            handler: this.getEnvironment,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['admin'],
+                    operationId: 'getEnvironment',
+                    responses: { 200: featureEnvironmentInfoResponse },
+                }),
+            ],
+        });
 
-        this.post(
```

Comment:
So previously the methods for any of mutational HTTP verbs would require an explicit permission, including a `NONE` which resolves to no permissions needed. We put that in because it turned out that we kept on forgetting to include permissions and that blocked us from making silly mistakes, resulting a compiler error if you didn't choose to explicitly set a `NONE` permission. This looks like this would remove that safety layer. Would it be possible to modify the IRouteOptions interface to not have permissions as a nullable?

Reply:
Yes this makes a lot of sense.  The routeOptions have been there a while, but this edit makes a lot of sense